tests/jerry/regression-test-issue-736.js

```diff
@@ -31,7 +31,7 @@ try {
 }
 
 try {
-  eval("var x; x instanceof assert;");
+  eval("var x = {}; x instanceof assert;");
```

Comment:
It is a related test, which fails after this patch since `undefined instanceof assert` does not throw error anymore. This is the correct behaviour. However, the throwing is tested here, so x needs to be an object.